var/spack/repos/builtin/packages/cmake/package.py

```diff
@@ -183,8 +185,16 @@ class Cmake(Package):
         depends_on('rhash', when='@3.8.0:')
 
     with when('+ownlibs'):
-        depends_on('openssl')
-        depends_on('openssl@:1.0', when='@:3.6.9')
+        def enable_openssl():
+            depends_on('openssl')
+            depends_on('openssl@:1.0', when='@:3.6.9')
+
+        if is_windows:
```

Comment:
what's the deal with all this `is_windows`? you should always check the target, not the host, we have platform=... for specs. All this is_windows stuff is so unreadable, please improve it.